content/en/serverless/google_cloud_run/_index.md

```diff
@@ -53,10 +78,12 @@ CMD ["ddtrace-run", "python", "app.py"] (adapt this line to your needs)
 See [Tracing Python Applications][1] for detailed instructions. [Sample code for a simple Python application][2].
```

Comment:
we should link to the "Instrument Your Application" section, because serverless-init already set up the agent?

Reply:
I like the idea. It also remove the duplicated per language code there (or it can be done as separate tabs in the other section)